thirdparty/runtime_environments.go

```diff
@@ -0,0 +1,61 @@
+package thirdparty
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/evergreen-ci/gimlet"
+	"github.com/pkg/errors"
+)
+
+type RuntimeEnvironmentsClient struct {
+	Client  *http.Client
+	BaseURL string
+	APIKey  string
+}
+
+func NewRuntimeEnvironmentsClient(baseURL string, apiKey string) *RuntimeEnvironmentsClient {
+	c := RuntimeEnvironmentsClient{
+		Client:  &http.Client{},
+		BaseURL: baseURL,
+		APIKey:  apiKey,
+	}
+	return &c
+}
+
+// getImageNames returns a list of strings containing the names of all images from the runtime environments api
+func (c *RuntimeEnvironmentsClient) getImageNames(ctx context.Context) ([]string, error) {
+	apiURL := fmt.Sprintf("%s/rest/api/v1/imageList", c.BaseURL)
+	request, err := http.NewRequestWithContext(ctx, http.MethodGet, apiURL, nil)
+	if err != nil {
+		return nil, err
+	}
+	request = request.WithContext(ctx)
+	request.Header.Add("Content-Type", "application/json")
+	request.Header.Add("Api-key", c.APIKey)
```

Comment:
this is the change Kim suggested:
```suggestion
	request.Header.Add("Api-Key", c.APIKey)
```